tools/aws_lab_setup/roles/control_node/tasks/main.yml

```diff
@@ -22,13 +22,13 @@
     - control
 
 - name: Install Ansible
-  - pip:
+  pip:
     name: ansible
-    
-  tags:
-    - control_node
-    - control
-  when: control_node_install_ansible
+    state: latest
+  #tags:
```

Comment:
Don't comment out code and leave it in a PR. I'm not a fan of the tagging in the provisioner especially since they are undocumented, but I prefer they come out together -- probably part of #141.